Prepare Getting Started with HtmlEditor (#2231)
DevExpress/devextreme-documentation: concepts/05 UI Components/HtmlEditor/00 Getting Started with HtmlEditor/07 Resize Media Content.md

> HtmlEditor can resize media embedded within its content. To enable this functionality pass the [enabled](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/mediaResizing/#enabled) property to the [mediaResizing](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/mediaResizing) configuration object and set it to **true**.

Reviewer: "HtmlEditor can resize media" - who does what?

```suggestion
Users can resize images embedded within the content. To enable this functionality, set the [mediaResizing](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/mediaResizing/).[enabled](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/mediaResizing/#enabled) property to **true**.
```

                enabled: true
            }
        });
    });

##### Angular

    <!-- tab: app.component.html -->
    <dx-html-editor...>
        <!-- ... --->
        <dxo-media-resizing
            [enabled]=true>
        </dxo-media-resizing>
    </dx-html-editor>

##### Vue

    <!-- tab: App.vue -->
        <template>
            <DxHtmlEditor...>
                <!-- ... -->
                <DxMediaResizing :enabled="true"/>
            </DxHtmlEditor>
        </template>
        <script>
        import {
        ...,
        DxMediaResizing,
        } from 'devextreme-vue/html-editor';

        export default {
            components: {
                ...,
                DxMediaResizing,
            },
            // ...
        };
        </script>

##### React

    <!-- tab: App.js -->
    // ...
    import HtmlEditor, {
    ...,
    MediaResizing,
    } from "devextreme-react/html-editor";

    App = () => {
        return (
            <>
                <HtmlEditor...>
                    {/* ... */}
                    <MediaResizing enabled={true} />
                </HtmlEditor>
            </>
        );
    }

    export default App;

    

> ---

Reviewer: Rename this topic to Resize Images
